tinymce-per-block/src/blocks/image-block/index.js

```diff
@@ -72,5 +72,13 @@ registerBlock( 'image', {
 			attrs: { /* caption: block.caption ,*/ align: block.align },
 			rawContent
 		};
+	},
+	create: () => {
+		return {
```

Comment:
>I have a personal preference over having it separate instead of handling an empty block as input. We could reconsider later.

That's fine, let's see how it works out 👍